chaosaws/ec2/actions.py

```diff
@@ -66,44 +71,48 @@ def stop_instances(instance_ids: List[str] = None, az: str = None,
             "To stop EC2 instances, you must specify either the instance ids,"
             " an AZ to pick random instances from, or a set of filters.")
 
+    if az and not instance_ids and not filters:
+        logger.warn("Based on configuration provided I am going to stop all
+                    instances in AZ {}. You have 15 seconds for
+                    Control-C if this is not what you intended".format(az))
+        sleep(15)
```

Comment:
I'm feeling uneasy about this. How can you automate? What is the rationale here?